scripts/build_scripts/build_zips.py

```diff
@@ -167,14 +186,25 @@ def get_targets_args(targets):
       camke args included targets.
   """
   result_args = []
+  
+  support_targets = SUPPORT_TARGETS
+  if is_tvos_build():
+    support_targets = TVOS_SUPPORT_TARGETS
+    # Explicitly disable Dynamic Links on tvOS. This is required because we're
+    # truncating support_targets to a subset, and the "target in support_targets"
+    # loop below won't append this property definition for us.
+    result_args.append("-DFIREBASE_INCLUDE_DYNAMIC_LINKS=OFF")
```

Comment:
I think it would make more sense to enforce this in the CMake logic, like how C++ handled it: https://github.com/firebase/firebase-cpp-sdk/blob/main/CMakeLists.txt#L120